packages/dashboard/src/vaadin-dashboard-layout-mixin.js

```diff
@@ -77,7 +86,7 @@ export const DashboardLayoutMixin = (superClass) =>
           --_vaadin-dashboard-row-height: minmax(var(--_vaadin-dashboard-row-min-height, auto), auto);
 
           display: grid;
-          overflow: auto;
+          overflow-x: hidden auto;
```

Comment:
```suggestion
          overflow: hidden auto;
```

Should you not be able to scroll overflowing Dashboard horizontally?

https://github.com/user-attachments/assets/bc6a1771-63e2-4804-a142-f0df371a05a0

Reply:
I added `overflow-x:hidden` to prevent a small horizontal scroll caused by the offset resize buttons with zero dashboard padding. I figured there'd only ever be a need for vertical scrolling, since the whole thing is responsive and collapses down to 1 col, as long as you set the min-col-width small enough for small viewports.

I suppose there is still a chance horizontal scroll could be needed, and now that I think about it, the resize-button offsets could be adjusted the same way as the section outlines... I'll give that a shot.